src/pages/getting-started/app-guides/app-authorization/index.mdx

```diff
@@ -7,12 +7,30 @@ tagline: "Find out how to implement an authorization flow and configure the App
 desc: "Find out how to implement an authorization flow and configure the App authorization building block."
 ---
 
+# Authorization for LiveChat apps
+
 If you want to build an integration that calls the LiveChat APIs, you'll need to implement an authorization flow and configure the **App authorization** building block.
 
 - <SectionLink to={"/authorization/authorizing-api-calls/"}>Read about different authorization flows</SectionLink>
 
 - <SectionLink to={"/authorization/authorization-in-practice/"}>Tutorial: Authorization in practice </SectionLink>
 
-<div style={{maxWidth: '600px'}}>
-  <img src="/images/getting-started/app-guides/app-authorization/livechat-authorization-block.png" alt="Authorizaton building block" width="845px" height="385px"/>
+<div style={{maxWidth: '350px'}}>
```

Comment:
I'm not sure about this image. It's very much out of context and I wonder if it brings any value 🤔 Besides, we no longer display BBs as tiles, but rather as a list

Reply:
I do not think it would lose anything if we got rid of it:)